app/lib/frontend/static_files.dart

```diff
@@ -27,25 +29,27 @@ String _resolveStaticDirPath() {
 }
 
 class StaticsCache {
-  final String staticPath;
+  final String staticPath = _defaultStaticPath;
```

Comment:
Now it's no longer possible to override this?

Reply:
I'm not sure if there is a need for the override, and if that arises, it should be easy to do.